language/tools/vm-genesis/src/lib.rs

```diff
@@ -27,24 +25,25 @@ use diem_types::{
         authenticator::AuthenticationKey, ChangeSet, Script, Transaction, WriteSetPayload,
     },
 };
-use diem_vm::{data_cache::StateViewCache, txn_effects_to_writeset_and_events};
+use diem_vm::{convert_changeset_and_events, data_cache::StateViewCache};
```

Comment:
Broader question: what is our plan for maintaining vm-genesis going forward?

Reply:
Great question. I don't think we have reached any consensus yet, but personally I feel like
1. The genesis itself may no longer be that important for a running blockchain, but incremental upgrades and the ability to generate them definitely are. Luckily @runtian-zhou has built a lot of great tools for this.
2. Some form of genesis will still be required by tests meaning that we'll need to keep at least some of the generation logic. However I think it may be possible for us to greatly simplify this, or develop some light weighted genesis generation library that is less specific about Diem. (See #5972 for example)